### ISSUE TITLE
Make onQueryCreated more robust with regards to creating resources from query Results

### ISSUE THREAD

**Issue by DavidBruant** [MyWebIntelligence/MyWebIntelligence#145]
Currently, it's a naive `db.Resources.create(queryResults)`. This will most certainly fail if there are already resources with these URLs.

At the very least, add tests for this case.

**Comment by DavidBruant** [comment]
Wrote the test. It fails indeed.
```
onQueryCreated error { [error: duplicate key value violates unique constraint "resources_url_key"]
(...)
  detail: 'Key (url)=(http://a.web/1) already exists.',
```

Fixing it right now.

**Comment by DavidBruant** [comment]
Fixed in #149

**Comment by DavidBruant** [comment]
Side note, this was a pretty **catastrophic bug**. 
If a query result had a single URL already known in the db, the resources wouldn't be created in the database (and no task would be created, no crawl, etc.)
